### PR TITLE
Implement Farmer model and associated Pydantic schemas for profile management

### LINKED ISSUE #68: Farmer Model + Profile Management
## Files:
	•	app/models/farmers/farmer.py
	•	app/schemas/farmer.py
	•	alembic/versions/002_create_farmers_table.py

## Tasks:
	•	Farmer model with relation to User
	•	Fields: farm_name, farm_size, location, organic_certified, description
	•	Schemas: FarmerCreate, FarmerUpdate, FarmerResponse
	•	Migration with foreign key to users

✅ Test Criteria:

```bash
# FARMER-type user can create a farmer profile  
# Profile is correctly linked to the User  
# Migration runs without errors  
```

### PR DESCRIPTION
**Pull request by WassCodeur**
# Pull Request RevolutionaryFarmers

## 📝 Summary

Provide a brief summary of what this PR accomplishes.

Implementing the Farmer model

This pull request introduces the Farmer model, allowing users of type “FARMER” to create and manage their farming profile.

---

### 🪧 Related Issues

- Closes #68.

---

### 🏁 Type of Change

Mark with an `x` all the checkboxes that apply (like `[x]`).

- [x] 📝 Documentation (updates to README, docs, or comments)
- [x] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] 👌 Enhancement (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)

---

### 🔄 Changes Made

Provide a clear and concise description of what you changed and why.

---

### 🚀 Implementation Details

Describe how you implemented the changes.

---

### 🛠 Technical Notes

Include any technical details that reviewers should be aware of.

---

### ✅ Tests Results

Describe the tests you performed to verify your changes:

---

### Test Coverage

- [ ] ✅ Unit Tests
- [ ] ✅ Integration Tests
- [ ] ✅ Manual Testing

---

### 📸 Evidence

Provide relevant evidence of testing (screenshots, test outputs, etc.).

---

### 🔍 Testing Notes

Include any special testing considerations or edge cases checked.

---

### 🔜 Next Steps

- Respond to review comments
Make sure you respond to all reviewer comments. If a change is suggested, make it and clearly indicate that the comment has been taken into account. If you choose not to make a change, briefly explain your decision.

- Check test coverage
Confirm that unit or integration tests exist for the Farmer model. If they don't, add them to validate the correct operation of the model and its associated schemas.

- Update documentation
If changes have been made to the API or database structure, update the corresponding documentation. This includes README files, usage examples and comments in the code.

- Ensure code style compliance
Checks that code complies with established style conventions (e.g. PEP8 for Python). Use automated tools such as Flake8 or Black to ensure compliance.

- Perform a security review
Analyzes changes to detect possible vulnerabilities, such as the exposure of sensitive data or potential flaws. Use static analysis tools to help you with this task.

- Update dependencies if necessary
If new libraries have been added, make sure they are necessary and don't introduce unnecessary dependencies. Use tools such as depcheck to identify dependencies that are not necessary.

- Take another issue in the repository

---

### CODE REVIEW

File: app/models/__init__.py [hunk 1]
```diff
@@ -21,8 +21,10 @@
 # from .products import Product, ProductCategory
 # from .orders import Order, OrderItem
 # from .shared import Location
+from .farmers import Farmer, FarmerVerification
```
[the remark this answers is not in the document]
Comment (aguilar1x) [reply app/models/__init__.py:24]: Remove FarmerVerification import
